src/docfx/restore/Restore.cs

```diff
@@ -79,7 +79,7 @@ private static void RestorePackages(ErrorBuilder errors, BuildOptions buildOptio
             ParallelUtility.ForEach(
                 errors,
                 GetPackages(config).Distinct(),
-                item => packageResolver.DownloadPackage(item.package, item.flags));
+                item => packageResolver.ResolvePackage(item.package, item.flags));
```

Comment:
>ResolvePackage [](start = 40, length = 14)

no need to change

Reply:
then remove the `DownloadPackage` from `localPackageResolver`

---
In reply to: [486173641](https://github.com/dotnet/docfx/pull/6531#discussion_r486173641) [](ancestors = 486173641)